pkg/cmd/admin/network/project_options.go

```diff
@@ -84,7 +84,14 @@ func (p *ProjectOptions) Validate() error {
 		errList = append(errList, errors.New("must provide --selector=<project_selector> or projects"))
 	}
 
-	// TODO: Validate if the openshift master is running with mutitenant network plugin
```

Comment:
I queried the history of `ClusterNetwork` API and found that it was added on [14 May 2015](https://github.com/openshift/origin/blob/b7d2fe8a360c0730c49f48de467180a78318bdad/pkg/client/client.go#L132). It means that an apiserver before that day would not support `ClusterNetwork` API. So the question is, do we expect the functionality to be able to run against such an old apiserver? :)